src/scripts.js

```diff
@@ -62,49 +66,57 @@ Promise.all([
-  $('#user-average-hours-slept').text(sleep.returnAverage('hoursSlept', userSleepData));
-  $('#user-current-step-count').text(activity.returnMetricByDate('numSteps', userActivityData, currentDate));
+  updateUserInfo();
+  updateWaterUserInfo();
+  updateSleepUserInfo();
+  updateActivityUserInfo();
+  updateFriendChallenge();
+}
+
+function updateUserInfo() {
+  let info = ['name', 'email', 'address', 'dailyStepGoal'];
+  info.forEach(el => {
+    $(`.user-inputs span[data-info=${el}]`).text(user[el]);
+  })
+}
+
+function updateWaterUserInfo() {
+    $('#user-water-by-day').text(hydration.returnMetricByDate('numOunces', userHydroData, currentDate));
+    $('#user-water-by-average').text(hydration.returnAverage('numOunces', userHydroData));
+    $('#user-water-trend-week').text(displayWaterStatus());
+}
+
+function updateSleepUserInfo() {
+  let info = ['hoursSlept', 'sleepQuality'];
+  info.forEach(el => {
+    $(`#user-sleep-day p[data-info=${el}]`).text(sleep.returnMetricByDate(el, userSleepData, currentDate));
+    $(`#user-average-sleep p[data-info=${el}]`).text(sleep.returnAverage(el, userSleepData));
+  });
   $('#user-rested').text(displaySleepStatus());
-  $('#user-current-mins-active').text(activity.returnMetricByDate('minutesActive', userActivityData, currentDate));
-  $('#user-current-miles-walked').text(activity.returnMilesWalkedByDate(userActivityData, user));
-  $('#user-current-step-count-vs-average').text(activity.returnAverage('numSteps', userActivityData));
-  $('#all-users-average-step-count').text(activity.returnAverage('numSteps'));
-  $('#user-current-stairs-climbed').text(activity.returnAverage('flightsOfStairs', userActivityData));
-  $('#all-users-average-stairs-climbed').text(activity.returnAverage('flightsOfStairs'));
-  $('#user-current-active-mins').text(activity.returnAverage('minutesActive', userActivityData));
-  $('#all-users-average-active-mins').text(activity.returnAverage('minutesActive'));
-  $('#user-step-count-by-week').text(activity.returnMetricByWeek('numsSteps', userActivityData))
-  $('#user-stairs-climbed-by-week').text(activity.returnMetricByWeek('flightsOfStairs', userActivityData))
-  $('#user-mins-active-by-week').text(activity.returnMetricByWeek('minutesActive', userActivityData))
-  // $('#winner-name').text(returnFriendChallengeWinner(user, activity.data))
-  $('#user-water-trend-week').text(displayWaterStatus());
+}
+
+function updateActivityUserInfo() {
+  let info = ['numSteps', 'minutesActive', 'flightsOfStairs'];
+  info.forEach(el => {
+    $(`#user-activity-day p[data-info=${el}]`).text(activity.returnMetricByDate(el, userActivityData, currentDate));
+    $(`.you-vs-world span[data-info="user-${el}"]`).text(activity.returnAverage(el, userActivityData));
+    $(`.you-vs-world span[data-info="all-${el}"]`).text(activity.returnAverage(el));
+  });
+  $('#user-activity-day p[data-info="miles"]').text(activity.returnMilesWalkedByDate(userActivityData, user));
```

Comment:
Thank you for breaking up these JQuery DOM manipulations. Makes it much easier to read